R-packages/modeltools/R/matrix.R

```diff
@@ -0,0 +1,75 @@
+#' Create a training and prediction matrices for a given ahead.
+#'
+#' @param lagged_df Data frame of lagged data.  It should have the following columns:
+#'     \itemize{
+#'     \item{`geo_value`}{Strings of geographic locations.}
+#'     \item{`time_value`}{Dates of training data.}
+#'     \item{Covariate columns}{Columns with names of the form `value-{days}:{signal}` whose values
+#'         correspond to `{signal}` `{days}` before `time_value`}
+#'     \item{Response columns}{Columns with names of the form `value+{n}:{response}` whose values
+#'         correspond to `{response}` `{n}` incidence period units after `time_value`.  Since
+#'         columns beginning with "value+" are interpretted as the response rather than a
```

Comment:
```suggestion
 #'         columns beginning with "value+" are interpreted as the response rather than a
```
sorry for the super nitpick